### ISSUE TITLE
Make metadata header separator configurable "~~~~~~" or support standards such as Jekyll front matter

### ISSUE THREAD

**Issue by sengopal** [jbake-org/jbake#231]
Make metadata header separator configurable "~~~~~~" or support standards such as Jekyll front matter

Reason: I am using Atom.io and markdown-preview-plus for drafting posts. The separator "~~~~~~" seems to render the markdown as incorrect and the preview fails.

<img width="1439" alt="screen shot 2015-10-12 at 4 59 57 pm" src="https://cloud.githubusercontent.com/assets/787381/10442557/54e4b164-7108-11e5-92d6-211e13220916.png">

Raised it on markdown-preview-plus as well - https://github.com/Galadirith/markdown-preview-plus/issues/134

**Comment by jangalinski** [comment]
the ~ is also hard to type on a macbook (ger) so +1 for making this more standard

**Comment by sengopal** [comment]
Jekyll head metadata is with ---------. Even replacing or accepting ---------- will also help.

**Comment by jonbullock** [comment]
Adding support for --------- should be straight forward to do. Have you tried a file with the JBake separator replaced with ---------  to check it works? 

As I thought Jekyll's YAML front matter starts with --- and ends with --- ?

**Comment by sengopal** [comment]
If i try with --------- , the file does not get picked up for conversion. Do you think Jekyll's format will work, i can test it out.

**Comment by jonbullock** [comment]
Yeah try the Jekyll format to confirm, adding support for the full front matter tags is a bit more work compared to changing the separator.
